platforms/minigame/platforms/taobao-mini-game/wrapper/fs-utils.js

```diff
@@ -23,14 +23,14 @@
  THE SOFTWARE.
  ****************************************************************************/
 const fs = my.getFileSystemManager ? my.getFileSystemManager() : null;
-const outOfStorageRegExp = /the maximum size of the file storage/;  // not exactly right
+const outOfStorageRegExp = "10028";  // not exactly right
 
 const fsUtils = {
 
     fs,
 
-    isOutOfStorage (errMsg) {
-        return outOfStorageRegExp.test(errMsg);
+    isOutOfStorage (errorCode) {
+        return errorCode.toString() == outOfStorageRegExp;
```

Comment:
It is wired to compare string content to check out of storage.

Reply:
taobao use errorCode 10028 to judge out of storage.

Reply:
What's the type of errorCode?

Reply:
normally it will be string

Reply:
In case of the error code in the other apis like donwloadFile or copyFile which will trigger LRU in the feature, will be not string type.

Reply:
- as the type is not fixed, so it is better to do some logic to make it more cearly
- please use `===` instead
- i think `outOfStorageRegExp` is not accurate, may be `outOfStorageErrorCode`